src/components/atoms/AvatarMenu/AvatarMenu.tsx

```diff
@@ -0,0 +1,62 @@
+import Avatar from '@material-ui/core/Avatar'
+import IconButton from '@material-ui/core/IconButton'
+import Menu from '@material-ui/core/Menu'
+import MenuItem from '@material-ui/core/MenuItem'
+import * as React from 'react'
+
+import { User } from 'interfaces/Props'
+
+interface AvatarMenuProps {
+  readonly user?: User
+  readonly onSignOut?: () => any
+}
+
+interface AvatarMenuState {
+  anchorEl: object
+}
+
+export class AvatarMenu extends React.Component<AvatarMenuProps, AvatarMenuState> {
+  state: { anchorEl: any } = {
```

Comment:
Shouldn't this be `state: AvatarMenuState`? 

Or maybe not type it at all — don't React typings resolve this by the `React.Component<AvatarMenuProps, AvatarMenuState>`?